scripts/xtensa-build-zephyr.py

```diff
@@ -311,18 +346,18 @@ def execute_command(*run_args, **run_kwargs):
 
 	return subprocess.run(*run_args, **run_kwargs)
 
-def symlink_or_copy(directory, origbase, newbase):
-	"""Create a symbolic link or copy in the same directory. Don't
-	bother Windows users with symbolic links because they require
-	special privileges. Windows don't care about /lib/firmware/sof/
-	anyway. Make a copy instead to preserve cross-platform consistency.
-
+def symlink_or_copy(targetdir, targetfile, linkdir, linkfile):
+	"""Create a relative symbolic link or copy. Don't bother Windows users
+	with symbolic links because they require special privileges.
+	Windows don't care about /lib/firmware/sof/ anyway.
+	Make a copy instead to preserve cross-platform consistency.
```

Comment:
Please don't mix whitespace changes with logical changes.

Reply:
@marc-hb, the functionality of `symlink_or_copy()` is changed, so the comments also needs to be updated. It is no longer limited to the same directory as it was before this change.

Reply:
Apologies I overestimated GitHub...